qos/evm/observe.go

```diff
@@ -0,0 +1,69 @@
+// evm package provides the support required for interacting
+// with an EVM blockchain through the gateway.
+package evm
+
+import (
+	"context"
+)
+
+// observationSet provides all the functionality required
+// by the message package's ObservationSet to handle the sharing
+// of QoS data between PATH instances, and the updating of local
+// PATH instance's QoS data on EVM endpoints.
+var _ message.ObservationSet = observationSet{}
+
+// observation captures the result of processing an endpoint's
```

Comment:
I was thinking of naming this `requestObservation` but there's nothing `request` related here. 

Is this intended to be something like `evm_GetHeight_Observation` for clarity?

Reply:
Yes, comment updated.